src/main/kotlin/no/nav/syfo/metric/Metrics.kt

```diff
@@ -91,6 +91,13 @@ val COUNT_PREDIKSJON_OUTPUT_FAILED: Counter = Counter.build()
     .help("Counts the number of failed queries for prediksjon_output")
     .register()
 
```

Comment:
Hva er grunnen til å ha både PREDIKSJON_GET_OUTPUT_FAILED og PREDIKSJON_GET_OUTPUT_ERROR (med lik help string) ? 
Hvis man vil skille på IllegalArgumentException og andre Exceptions så bør tellerne kanskje være mer beskrivende. Jeg ville i hvert fall lurt på hvilken teller som brukes til hva... Med forbehold om at jeg ikke kjenner godt til hvordan man setter opp slike metrics :)

Reply:
Skal forsøke å presisere beskrivelsene. Men jeg tror det er et poeng og ha to tellere slik at man kan skille mellom feil som skyldes feil input (400-feil) og feil som skyldes et uventet feilsituasjon i tjenerkoden (500-feil, feks at databasen ikke svarer). Denne siste feiltypen ble ikke telt opp, og vi trenger det for å kunne vise antall 500-feil i Grafana.

Reply:
Ja, det gir mening. Bedre beskrivelse blir bra 👍